archlinux/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM lopsided/archlinux:latest
```

Comment:
While the release.yml for Debian packages still uses the project root's `/debian/` directory, that one is going away very soon (thanks for reminding me), as I want to reduce redundancies in package creation by using the same logic for release builds that has been used for regular test builds.

Since this directory will be solely used by the CI, what about moving this directory into `/.github/archilinux/` instead?

Reply:
Yea I will move it in there as that sounds like a good plan.